R-package/R/lgb.cv.R

```diff
@@ -550,7 +550,7 @@ lgb.stratified.folds <- function(y, k) {
 
       ## Create a vector of integers from 1:k as many times as possible without
       ## going over the number of samples in the class. Note that if the number
-      ## of samples in a class is less than k, nothing is producd here.
+      ## of samples in a class is less than k, nothing is produced here.
```

Comment:
Just out of my curiosity, why double `##` here?

Reply:
no reason at all! According to the git blame, these lines were added 5 years ago

https://github.com/microsoft/LightGBM/blame/0df38106b48161e12204de5621e406cc6c19a3e6/R-package/R/lgb.cv.R#L512-L519

I'd support a PR changing them (and other instances like this) to a single `#`

Reply:
OK, got it! 🙂